### PR TITLE
Update list_slice #8087 and add list_reverse #8477

### PR DESCRIPTION
**Pull request by maiadegraaf**
- duckdb/duckdb#8087
- duckdb/duckdb#8477

### CODE REVIEW

File: docs/sql/functions/nested.md [hunk 1]
```diff
@@ -10,62 +10,67 @@ In the descriptions, `l` is the three element list `[4, 5, 6]`.
...
+| `array_slice(`*`list`*`, `*`begin`*`, `*`end`*`, `*`step`*`)` | Alias for `list_slice` with `step`.                                                                                                                                                | `array_slice(l, 1, 3, 2)`                | `[4, 6]`          |
+| `list_reverse(`*`list`*`)`                                    | Reverses the list.                                                                                                                                                                 | `list_reverse(l)`                        | `[6, 5, 4]`       |
+| `array_reverse(`*`list`*`)`                                   | Alias for `list_reverse`.                                                                                                                                                          | `array_reverse(l)`                       | `[6, 5, 4]`       |
```
Comment (Mause) [inline docs/sql/functions/nested.md:26]: Can we add an alias column instead of double listing functions?

Comment (maiadegraaf) [reply docs/sql/functions/nested.md:26]: Agreed, it makes the table cluttered.

File: docs/sql/functions/nested.md [hunk 2]
```diff
@@ -10,62 +10,35 @@ In the descriptions, `l` is the three element list `[4, 5, 6]`.
...
-| `list_resize(`*`list`*`, `*`size`*`[, `*`value`*`])`  | Resizes the list to contain `size` elements. Initializes new elements with `value` or `NULL` if `value` is not set.                                                             | `list_resize([1,2,3], 5, 0)`             | `[1, 2, 3, 0, 0]` |
-| `array_resize(`*`list`*`, `*`size`*`[, `*`value`*`])` | Alias for `list_resize`.                                                                                                                                                        | `array_resize([1,2,3], 5, 0)`            | `[1, 2, 3, 0, 0]` |
+| Function                                                     | Alias                                             | Description                                                                                                                                                                        | Example                                 | Result            |
```
Comment (Mause) [inline docs/sql/functions/nested.md:13]:
```suggestion
| Function                                                     | Aliases                                           | Description                                                                                                                                                                        | Example                                 | Result            |
```

**Review by Mause — APPROVED** [review]
Looks good! Just one tiny fix then we can merge

**Comment by Mause** [comment]
Thanks!

**Pull request merged**